django_app/report_a_suspected_breach/templates/report_a_suspected_breach/tasklist.html

```diff
@@ -24,7 +24,7 @@ <h1 class="govuk-heading-l">Report a suspected breach of trade sanctions</h1>
                         {% endif %}
                     </div>
                     <div
-                        class="govuk-task-list__status {% if not task.can_start %}govuk-task-list__status govuk-task-list__status--cannot-start-yet{% endif %}"
+                        class="govuk-task-list__status {% if not task.can_start and task.status_text != 'Completed' %}govuk-task-list__status govuk-task-list__status--cannot-start-yet{% endif %}"
```

Comment:
Awesome work! Wondering if this might also fix? So we can be sure that we're changing the colour for the incomplete section only:
```suggestion
                        class="govuk-task-list__status {% if task.status_text == 'Cannot start yet' %}govuk-task-list__status govuk-task-list__status--cannot-start-yet{% endif %}"
```
If not, feel free to merge.

Reply:
Agreed, but furthermore I think we should remove the duplicate class:

```suggestion
                        class="govuk-task-list__status {% task.status_text == 'Cannot start yet' %}govuk-task-list__status--cannot-start-yet{% endif %}"
```

Reply:
We should also add some front-end tests to check for this, I'll run you through this in the catchup